br-webui/_includes/header.liquid

```diff
@@ -52,9 +52,15 @@
 				<form class="navbar-form navbar-right">
 					<button onclick="reboot()" class="btn btn-primary btn-sm navbar-btn">Reboot</button>
 					<button onclick="shutdown()" class="btn btn-error btn-sm navbar-btn">Shutdown</button>
+                <ul class="toggle">
+                    <label class="switch">
+                        <input id='advanced-checkbox' type="checkbox">
+                        <span class="slidernew round"></span>
+                    </label>
+                    Enable Advanced Options
+                </ul>
 				</form>
 			</div><!--/.nav-collapse -->
-		</div>
 	</nav>
-
+	<div class="heading"></div>
```

Comment:
this does not appear to be necessary. I see no difference when I remove it.

Reply:
oh yes. I should have removed that. that was for an old commit.